maps/kibana/layers/10mriverslakecenterl/1b70dba2-10m-rivers-lake-centerlines.dbf

```diff
@@ -0,0 +1 @@
+/Users/cjcenizal/Documents/MapBox/cache/1b70dba2-10m-rivers-lake-centerlines/1b70dba2-10m-rivers-lake-centerlines.dbf
```

Comment:
This didn't cause an issue when I opened them up for some reason, but these all point to your local install...